presto-product-tests/src/main/java/com/facebook/presto/tests/ImmutableLdapObjectDefinitions.java

```diff
@@ -31,6 +31,8 @@
     private static final String americaOrgDistinguishedName = format("ou=America,%s", domain);
     private static final String asiaOrgDistinguishedName = format("ou=Asia,%s", domain);
     private static final String password = "LDAPPass123";
+    private static final String memberOf_relation = "memberOf";
+    private static final String member_relation = "member";
```

Comment:
Please capitalize these strings (and all previous static final strings as well). Perhaps you can also drop the "_relation" from the name.